app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+class User < ActiveRecord::Base
+
+  has_many :pictures
+  has_many :comments
+  has_many :likes
+
+  devise :omniauthable, :omniauth_providers => [:facebook]
+
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
+
+  def self.from_omniauth(auth)
+    where(provider: auth.provider, uid: auth.uid).first_or_create do |user|
+      user.email = auth.info.email || "#{auth.uid@facebook.com}"
```

Comment:
Lint/SpaceBeforeFirstArg: Put space between the method name and the first argument.